plugins/datadog/app/models/datadog_notification.rb

```diff
@@ -8,10 +8,10 @@ def initialize(deploy)
     @stage = @deploy.stage
   end
 
-  def deliver
+  def deliver(additional_tags: [], now: false)
     Rails.logger.info "Sending Datadog notification..."
 
-    status = @deploy.succeeded? ? "success" : "error"
+    status = @deploy.active? || @deploy.succeeded? ? "success" : "error"
```

Comment:
wut ... status is "success" when it's active ?

Reply:
Would `info` be more appropriate?